pants.toml

```diff
@@ -0,0 +1,28 @@
+[GLOBAL]
+pants_version = "2.10.0.dev2"
+backend_packages = [
+  "pants.backend.codegen.protobuf.python",
+  "pants.backend.experimental.codegen.thrift.apache.python",
+  "pants.backend.python",
+]
+
```

Comment:
I think you'll want to set `use_deprecated_python_macros = false`, then update lines 21 and 26 of this file to use `#` instead of `:` for the addresses.

Reply:
Fixed.